src/webview_provider/newLineagePanel.ts

```diff
@@ -297,12 +297,10 @@ export class NewLineagePanel implements LineagePanelView {
         // Flagging events where columns fetched from db are less than the number of columns in the manifest
         this.telemetry.sendTelemetryEvent("columnLineagePossibleStaleSchema");
       }
-      return true;
     } catch (exc) {
-      const error = String(exc);
-      window.showErrorMessage("Cannot add columns from DB: " + error);
-      throw new Error(error);
+      window.showErrorMessage("Cannot add columns from DB: " + exc);
     }
+    return true;
```

Comment:
@AdiGajbhiye : Shouldn't the exception clause not return false, and return true should be within the try block?

Reply:
@mdesmet yes you are right, we should return false, but upon doing so we are just [returning here](https://github.com/AltimateAI/vscode-dbt-power-user/blob/9e2afe35858dbb522b19ff89ddfecde6551a15a8/src/webview_provider/newLineagePanel.ts#L491), so the panel goes blank. might result in some breaking changes, so I kept this change for now
![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/31448917/b915ed59-eb56-4441-aaf4-7653cdbee6bd)

Reply:
@AdiGajbhiye : Can you lean in on this one?